src/prefect/agent/local/agent.py

```diff
@@ -22,8 +24,17 @@ class LocalAgent(Agent):
     def __init__(self, base_url: str = None, no_pull: bool = None) -> None:
         super().__init__()
 
-        base_url = base_url or "unix://var/run/docker.sock"
-        self.docker_client = docker.APIClient(base_url=base_url, version="auto")
+        if platform == "win32":
+            default_url = "npipe:////./pipe/docker_engine"
+        else:
+            default_url = "http+unix:///var/run/docker.sock"
+
+        # Determine Daemon URL
+        self.base_url = base_url or context.get("base_url")
```

Comment:
This is nice because it means we can set this via env var as well

Reply:
If you just add `or default_url` to the very end this would save you the two lines of code down below.  Or `context.get("base_url", default_url)`